packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "7fdb5cb179a15a63240590a2c3e0a6cdbdab89b5",
-    "ref_origin" : "dcos-mesos-1.2.x-f8706e5"
+    "ref": "bd983e54c7d941db4a67c1d5659a60014b3fec71",
+    "ref_origin" : "dcos-mesos-1.2.x-13a0782"
```

Comment:
It's deceptive to call this `dcos-mesos-1.2.x-13a0782` since there is no `13a0782` SHA anywhere in Apache Mesos, especially not in the (no longer existent) 1.2.x branch.
Please name the `mesosphere/mesos` branch `dcos-mesos-1.2.3` (using the latest tag or branch-SHA from Apache) and then update https://github.com/dcos/dcos/blob/1.9/packages/mesos/README.md to add your new backport SHAs
See https://github.com/dcos/dcos/blob/1.8/packages/mesos/README.md#critical-backports-from-mesos-11x-and-12x-branches for some example backport README content